src/backend/functions/work-packages-create.ts

```diff
@@ -31,32 +31,66 @@ export const createWorkPackage: Handler<FromSchema<typeof inputSchema>> = async
   } = body;
   // get the corresponding project so we can find the next wbs number
   // and what number work package this should be
+  const { carNumber, projectNumber, workPackageNumber } = projectWbsNum;
+
+  const wbsElem = await prisma.wBS_Element.findUnique({
+    where: {
+      wbsNumber: {
+        carNumber,
+        projectNumber,
+        workPackageNumber
+      }
+    }
+  });
+
+  if (wbsElem === null) throw new TypeError('No corresponding WBS Element for WBS Number.');
+
   const project = await prisma.project.findUnique({
```

Comment:
I think this query is unneeded because we can just add something like "include: project: true" to the lookup via WBS number. Then we'd just want to add a check before that to ensure that the supplied work package number is 0 to ensure that resulting project field on the WBS element will not be null